views/js/component/terminateExecutions/terminateExecutions.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+/**
+ * @author Oleksandr Khitko <oleksandr@taotesting.com>
+ */
+define([
+    'i18n',
+    'lodash',
+    'core/dataProvider/request',
+    'ui/component',
+    'tpl!taoSync/component/terminateExecutions/notification',
+], function (__, _, request, component, listTpl) {
+    'use strict';
+
+    /**
+     * Component that lists all the executions which are not in completed state and allows to terminate them
+     *
+     * @param {Object} config
+     * @returns {terminateExecutionsDialog}
+     */
+    function terminateExecutionsDialogFactory($container, config) {
+        var terminateExecutionsDialog;
+
+        if (!_.isPlainObject(config)) {
+            throw new TypeError('The configuration is required');
+        }
+
+        if (config.terminateUrl == undefined) {
+            throw new TypeError('terminateUrl configuration parameter is required');
+        }
+
+        if (config.csrfToken == undefined) {
+            throw new TypeError('CSRF token is missing');
+        }
+
+        function terminateDeliveryExecutions() {
+            var requestData = {
+                executionsId: config.activeExecutions
+            };
+            requestData[config.csrfToken.name] = config.csrfToken.token;
```

Comment:
I prefer to use `providers` for this

Reply:
but you can ask @krampstudio or @jsconan to know if it is required

Reply:
@jsconan sayid that it's "nice to have" to implement data provider, but in this case it's not really crucial